### DISCUSSION TITLE
Favicon not working

### DISCUSSION THREAD

**Discussion by mathisgauthey** [CaiJimmy/hugo-theme-stack#841] [Q&A] [answered]
Hi there, I'm sorry but I can't seem to make the favicon works.

What is the static folder ? I used the template file from github but I don't have any.

I used the assets folder for creating icons for social medias and my avatar, so I put my favicon here as well.

But I can't seem to find a way to make it work : 

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/46576952/9987e04d-2614-41da-acf4-0b6ec540866f)

**Comment by rmdhnreza** [comment]
Please move your icon to the `static` folder.

**Reply by mathisgauthey** [reply]
Hi, I did. I forgot about using the theme quick-start which doesn't match with the doc.

I'm using a git submodule now.

Favicon works by just forking the theme repo to a folder, puting the icon into static and inside config.

But using the theme as a submodule, the icon doesn't work :
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/46576952/7ef470d5-7407-4b3c-9c95-8a6257928e23)

There might be something wrong with the way I'm trying to update the configs. The issue is that configs are in TOML format inside the quickstart, in YAML format when I look at the git repo, TOML format in the example site of the repo, TOML format in the docs, but the only thing that work is by using the official repo and YAML for the favicon.

**Reply by mathisgauthey** [reply]
I did a hugo new site, added the theme as a git submodule from a fork of the theme I used.

Now which settings file should I use ? There's three of them, I guess the example site isn't usefull and I could delete it.

But my changes done on the HUGO.toml doesn't reflect on my site even though I copied them from the stack-theme-quick-start repo which has .toml configs.

In the same way, modifying the config.yaml of the theme directory doesn't do anything.

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/46576952/36617f55-75c8-4753-8281-4fc8b45e439f)

**Reply by rmdhnreza** [reply]
It's strange, open DevTools to see what error you get.

**Reply by mathisgauthey** [reply]
I'm sorry but I can't figure what part of dev tools from firefox you intends to use. I'm really new to web dev, only done C, C++ and Python scripting before, I'm a bit lost for now.

**Comment by CaiJimmy** [comment]
I've updated the starter template to include a sample favicon. Please take a look at it. I hope this is less confusing now.

**Reply by mathisgauthey** [reply, marked as answer]
Hi there, thanks for your time. I don't know if I'm being right but it seems the quickstart doesn't have the same level of feature the main theme have if I copy it inside my theme folder, hence I'll prefer using this way.

I think the issue is more related to how to change configs. 

Following the Hugo doc, I created a git repo, created a new hugo site, put my hugo stuff inside my repo.
Then I created a submodule for your theme.
Then I copied the example site into my base dir.
Now I deleted the HUGO.toml and figured the site not changing was because of the language section of the config.
I now try using config.yaml but the favicon isn't showing : 

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/46576952/6bdd8c4d-4c2e-4f4b-962a-c0c3cb24a2d5)

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/46576952/4f5902ed-e040-43da-83ef-edc70c3684d7)

Once I understand things better, I'd be glad to help for the documentation somehow.

**Reply by mathisgauthey** [reply]
It seems it needed a few hugo server restarts and a CTRL+SHIFT+R to force refresh on Firefox, I now have my local favicon working.

I'm still confused about where to put settings and change them but might be able to sort things out now !
